ansible/roles/node/defaults/main.yml

```diff
@@ -14,3 +14,5 @@ kube_node_binaries:
 kube_node_rpms:
   - kubernetes-client
   - kubernetes-node
+
+kubelet_args: ""
```

Comment:
We should rename the ``kubelet_args`` to ``kubelet_options`` with some example and comment it out. E.g.

```yaml
// kubelet_options:
// - "--host-network-sources=*"
```